az-digital/az_quickstart
# Closes #1348: Remove deprecated ParagraphsBehavior migrate process plugin.

## Description
Hoping to deprecate this plugin in favor of the one here: https://github.com/az-digital/az_quickstart/pull/1345/files#diff-df90683b10b3f30addedcf7829163c29bf81d810abf564c20ae743018e3c825a

If deprecated in 2.1.3 we can remove in 2.2 or 2.3

## Related Issue
Close #1348 
Blocked by #1345 

## How to test:
Since we are removing a plugin, we need to make sure that all of our migrations still work.
Luckily we have automated tests for most or all of the migrations
There should not be any errors in probo during quickstart install
There should not be any errors in probo during the migration test
There should not be any errors in probo during phpunit test
Text area page should have all paragraph behavior settings match modules/custom/az_demo/data/az_demo_text_paragraph.json
Text with background page should have all paragraph behavior settings match modules/custom/az_demo/data/az_demo_text_background_paragraph.json

## Types of changes

## Checklist:
